src/drivers/distance_sensor/tfmini/TFMINI.cpp

```diff
@@ -245,7 +245,7 @@ TFMINI::Run()
 	// perform collection
 	if (collect() == -EAGAIN) {
 		// reschedule to grab the missing bits, time to transmit 9 bytes @ 115200 bps
-		ScheduleClear();
+		//ScheduleClear();
```

Comment:
This affects all users of the sensor - could you elaborate on this?

Reply:
https://github.com/PX4/Firmware/pull/14423 @bkueng solved this problem actually.
I found this problem during updating our boards.
Will test and will PR again.